src/main/java/io/confluent/connect/hdfs/orc/OrcRecordWriterProvider.java

```diff
@@ -99,6 +103,24 @@ public void preFooterWrite(OrcFile.WriterContext writerContext) {
             );
           }
         } catch (IOException e) {
+          if (e instanceof RemoteException || e instanceof AlreadyBeingCreatedException) {
+            try {
+              /*
+               The current version of OrcFile.WriterOptions does not supports file overwriting
+               therefore deleting tmp file whenever`AlreadyBeingCreatedException` is thrown.
+                */
+              FileSystem fileSystem = FileSystem.get(
+                  URI.create(conf.getUrl()),
+                  conf.getHadoopConfiguration()
+              );
+              fileSystem.delete(path, true);
```

Comment:
Not too familiar with HDFS. Why are we deleting the file for ORC here? Why is `schema=null` not enough like with AVRO

Reply:
The current version of `org.apache.hadoop.hive.ql.io.orc.OrcFile.WriterOptions` does not supports file overwriting therefore deleting tmp file whenever`AlreadyBeingCreatedException` is thrown.